Automattic/wp-calypso
# App Banner: Migrate Images to Webpack

#### Changes proposed in this Pull Request

This migrates the images used only for the App Banner to Webpack. For details on the reasoning behind this, see #36850.

#### Testing instructions

1. Remove this line
https://github.com/Automattic/wp-calypso/blob/1a9fe203710351d36869b39500c7da9d36e62a9f/client/blocks/app-banner/index.jsx#L158

2. Visit the Reader, Stats, the Editor and the Notifications

3. Verify the image for the App Banner still appears

<img width="613" alt="Screenshot 2019-11-16 at 19 28 43" src="https://user-images.githubusercontent.com/43215253/68998341-e5bb3280-08a8-11ea-8cb5-61ae293f089c.png">
<img width="488" alt="Screenshot 2019-11-16 at 19 41 56" src="https://user-images.githubusercontent.com/43215253/68998361-2a46ce00-08a9-11ea-94c4-dc9ae2e4c3c4.png">

cc @vindl, @gwwar
